### DISCUSSION TITLE
Added HTML Card in HACS, but can't add a new Card of that type

### DISCUSSION THREAD

**Discussion by gmarchment** [PiotrMachowski/Home-Assistant-Lovelace-HTML-Jinja2-Template-card#25] [Q&A] [answered]
As title, I've added the component in as a HACS install, and can see the files in www/community.  When I go to add a new Card in the HA dashboard, I don't get a corresponding type.

Any ideas?

**Comment by PiotrMachowski** [comment]
You have to add it using yaml editor

**Reply by gmarchment** [reply]
Thanks.  I've tried, but get this.

<img width="468" alt="image" src="https://github.com/PiotrMachowski/Home-Assistant-Lovelace-HTML-Jinja2-Template-card/assets/151647330/3aa93871-82c0-401d-a7c6-44eafbda0fa6">

**Reply by PiotrMachowski** [reply]
It's it added in resources?

**Comment by gmarchment** [comment]
Yep - 
<img width="716" alt="image" src="https://github.com/PiotrMachowski/Home-Assistant-Lovelace-HTML-Jinja2-Template-card/assets/151647330/ea96a73b-0b7c-4486-a66d-8f296cc46543">

**Reply by PiotrMachowski** [reply]
Not here, in "resources" section in dashboard's config (available in menu under three dots)

**Reply by gmarchment** [reply]
Yep - it's there too.  (ps - thanks for your help!)

<img width="1247" alt="image" src="https://github.com/PiotrMachowski/Home-Assistant-Lovelace-HTML-Jinja2-Template-card/assets/151647330/ea2217b6-ca34-4659-bbab-8458539cd4e0">

**Reply by PiotrMachowski** [reply, marked as answer]
Ok, I see it now. You have installed a different card:
https://github.com/PiotrMachowski/lovelace-html-card

**Reply by gmarchment** [reply]
Thanks.  All sorted.

**Reply by workstations-ac** [reply]
> Thanks. All sorted.

Sorry to bug you, but could you let me know how you sorted this?  I have the correct repository linked and card added, but it doesn't show up in my dashboard cards.

**Reply by PiotrMachowski** [reply]
@workstations-ac this card is not shown in the list of available cards. You just have to configure it manually using yaml
